util/include/DigestImpl.ipp

```diff
@@ -22,31 +22,47 @@
 
 namespace concord::util::digest {
 
-class DigestUtil {
- public:
-  static size_t digestLength();
-  static bool compute(const char* input, size_t inputLength, char* outBufferForDigest, size_t lengthOfBufferForDigest);
-
-  class Context {
-   public:
-    Context();
-    void update(const char* data, size_t len);
-    void writeDigest(char* outDigest);  // write digest to outDigest, and invalidate the Context object
-    ~Context();
-
-   private:
-    void* internalState;
-  };
-};
-
 class DigestCreator {
  public:
   virtual ~DigestCreator() = default;
 
   virtual void init() = 0;
-  virtual void compute() = 0;
-  virtual void update() = 0;
-  virtual void final() = 0;
+  virtual void update(const char* data, size_t len) = 0;
```

Comment:
Should we add the following in this interface:
size_t digestLength() = 0;
bool compute(const char* input, size_t inputLength, char* outBufferForDigest, size_t lengthOfBufferForDigest) = 0;

Reply:
Added.